Tasks/GradleV2/task.json

```diff
@@ -211,6 +211,7 @@
             "visibleRule": "javaHomeSelection = JDKVersion",
             "options": {
                 "default": "default",
+                "1.17": "JDK 17",
```

Comment:
Could you please update the GradleV3 as well?

https://github.com/microsoft/azure-pipelines-tasks/blob/d883a26183ff56b5f43750f0e9a2ceae65ca4e9f/Tasks/GradleV3/task.json#L210-L218

Could you share some information about the testing that was done?

Additionally, please double-check that all related Gradle plugins (e.g. SonarQube, SqGradle, FindBugs, etc.) are compatible with Java 17.